### PR TITLE
add welcome-to-jsk program to apps

### PR DESCRIPTION
**Pull request by 708yamaguchi**
Add welcome-to-jsk program to apps.
Please merge this with https://github.com/jsk-ros-pkg/jsk_demos/pull/1242 if demo on 5/26(Sat) succeed.

### CODE REVIEW

**Comment by 708yamaguchi** [comment]
Now, We can use this demo from our smartphone.


@k-okada Could you please merge this with https://github.com/jsk-ros-pkg/jsk_robot/pull/935 (add app usage to readme) and https://github.com/jsk-ros-pkg/jsk_demos/pull/1242 (update program), if there is no problem?

screenshot_20180602-130253

**Comment by k-okada** [comment]
can you take good photo for this App? Let's imagine when you create another app. we car not use some photo for that app.

**Comment by k-okada** [comment]
may be this photo is good for hello world example

**Comment by 708yamaguchi** [comment]
thanks to @pazeshun

File: jsk_fetch_robot/jsk_fetch_startup/apps/welcome_to_jsk/welcome_to_jsk.app [hunk 1]
```diff
@@ -0,0 +1,9 @@
+display: Welcome to jsk
+platform: fetch
+launch: jsk_fetch_startup/welcome_to_jsk.xml
+interface: jsk_fetch_startup/welcome_to_jsk.interface
+<<<<<<< HEAD
```
Comment (pazeshun) [inline jsk_fetch_robot/jsk_fetch_startup/apps/welcome_to_jsk/welcome_to_jsk.app]: Don't you resolve conflict?

**Comment by 708yamaguchi** [comment]
@pazeshun 
Thank you for your review. I resolved the conflict.

**Pull request merged**